lib/actions/scan.coffee

```diff
@@ -32,6 +32,20 @@ dockerVersionProperties = [
 	'ApiVersion'
 ]
 
+scanErrorMessage = 'Could not find any balenaOS devices in the local network.'
+
+winScanErrorMessage = scanErrorMessage + """
+\n
+Note for Windows users:
+  The 'scan' command relies on the Bonjour service.
+  You can download Bonjour for Windows (included with Bonjour Print Services)
```

Comment:
This message will also be printed for users who already have Bonjour installed, so maybe it could invite users to check whether Bonjour is installed before re-installing it. Something like:
```
The 'scan' command relies on the Bonjour service. Check whether Bonjour
is installed (Control Panel > Programs and Features). If not, you can
download Bonjour for Windows ...
```